### PR TITLE
Release to PuppetForge

### PR DESCRIPTION
**Pull request by ppanero**
Tag bump for release
Changelog created

### CODE REVIEW

**Comment by traylenator** [comment]
@ppanero has no permission to tag so once merged a tag will need to be added as things stand.

File: metadata.json [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
   "license": "Apache-2.0",
   "source": "https://github.com/voxpupuli/puppet-misp",
   "project_page": "https://github.com/voxpupuli/puppet-misp",
   "issues_url": "https://github.com/voxpupuli/puppet-misp/issues",
   "dependencies": [
-    {"name":"puppetlabs-stdlib","version_requirement":">= 1.0.0 < 5.0.0"},
-    {"name":"puppetlabs-vcsrepo","version_requirement":">= 1.3.2 < 2.0.0"}
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 1.0.0 < 5.0.0"
```
Comment (bastelfreak) [inline metadata.json]: please bump the 1.0.0 to 4.13.1. That is the first version that works properly with puppet 4

**Comment by bastelfreak** [comment]
our rake task, that every collaborator can execute, will automatically create the tag.

File: metadata.json [hunk 2]
```diff
@@ -1,15 +1,21 @@
 {
   "name": "puppet-misp",
-  "version": "0.2.0",
+  "version": "0.2.3-rc0",
```
Comment (bastelfreak) [inline metadata.json]: bump it to the next version. I guess that will be 0.2.2 (thats mentioned in the CHANGELOG.md)? Please don't add the -rc0.

**Comment by bastelfreak** [comment]
Please also add a version requirement for Puppet to the metadata.json:

```json
  "requirements": [
    {
      "name": "puppet",
      "version_requirement": ">= 4.6.1 < 5.0.0"
    }
  ]
```

File: metadata.json (hunk 2 above)
Comment (ppanero) [inline metadata.json]: That was dont by the travis_release. I understood that I could executre the rake but its true I didnt have permsisions.

**Comment by bastelfreak** [comment]
ah okay. You have to update the changelog + bump the version in the metadata.json to the desired one. after that I will merge the PR and execute the rake task. this will create the tag + bump the version afterwards, to -rc0. We have a check implemented that prevents us from releasing -rc0 versions.

**Comment by ppanero** [comment]
@bastelfreak  Updated with puppet version requirement. Tho we want to accept 3.8.

**Pull request merged**